bypros/proxy.go

```diff
@@ -303,7 +311,7 @@ func (s *Service) Unfollow(req *Unfollow) (*EmptyReply, error) {
 func (s *Service) Query(req *Query) (*QueryReply, error) {
 	res, err := s.storage.Query(req.Query)
 	if err != nil {
-		return nil, xerrors.Errorf("failed to query: %v", err)
+		res = []byte(xerrors.Errorf("ERROR: failed to query: %v", err).Error())
```

Comment:
can't you catch it if it returns an error?

Reply:
when using the JavaScript library you will only get something like "the connection closed abnormally", but in this case I need the error message, which can indicate a wrong SQL syntax.

Reply:
OK, thanks - I thought that has been fixed in the meantime :(